src/include/enclave/enclave_state.h

```diff
@@ -51,8 +53,15 @@ typedef struct sgxlkl_enclave_state
     /* Memory shared with the host */
     sgxlkl_shared_memory_t shared_memory;
 
-    /* This flag is used by the tracing macros */
-    bool verbose;
+    /* Flags to track whether tracing macros are currently enabled (initialized
+     * according to the settings in `config`; the respective traces may be
+     * disabled temporarily and this flags track the current state) */
+    struct
+    {
+        bool verbose;
+        bool lkl_syscall;
+        bool internal_syscall;
```

Comment:
Are these the only ones that we have left or are there others?

Reply:
These are the only ones left at thus time, but if the tracing infrastructure gets updated, I can imagine more will be added.

Reply:
Don't we also have unsupported and overridden syscalls?

Reply:
Not sure what you're referring to. We have other traces, but those never get en/disabled after the config is sealed, so we don't need to track their state.